cmd/chasquid-util/chasquid-util.go

```diff
@@ -269,3 +271,56 @@ func domaininfoRemove() {
 		Fatalf("Error removing file: %v", err)
 	}
 }
+
+// chasquid-util alias-add <source> <target>
+func aliasAdd() {
+	source := args["<source>"].(string)
+	target := args["<target>"].(string)
+
+	sourceUser, sourceDomain := envelope.Split(source)
+
+	// ensure the domain exists
+	if _, err := os.Stat(filepath.Join(configDir, "domains", sourceDomain)); os.IsNotExist(err) {
+		Fatalf("Domain doesn't exist")
+		return
+	}
+
+	conf, err := config.Load(configDir + "/chasquid.conf")
+	if err != nil {
+		Fatalf("Error reading config")
+		return
+	}
+	os.Chdir(configDir)
+
+	// setup alias resolver
+	r := aliases.NewResolver()
+	r.SuffixSep = conf.SuffixSeparators
+	r.DropChars = conf.DropCharacters
+
+	r.AddDomain(sourceDomain)
+	aliasesFilePath := filepath.Join("domains", sourceDomain, "aliases")
+	if err := r.AddAliasesFile(sourceDomain, aliasesFilePath); err != nil {
+		Fatalf("%s: error loading %q: %v", sourceDomain, aliasesFilePath, err)
+		return
+	}
```

Comment:
This is duplicating the aliases-resolve from above, would you mind moving it to a helper function?

Reply:
As discussed on IRC, this is not a good suggestion as the duplication is small and isn't worth the readability loss of the helper function, sorry about that!